source/host/KTestSystem.cc

```diff
@@ -350,6 +350,8 @@ bool KTestSystem::runTest() {
         calc();
     }
 
+    // TODO: Use a system heap! std::string relies on heap allocation
+    m_sceneMgr->currentScene()->heap()->enableAllocation();
```

Comment:
Do you want to disable allocation after writing the string?

Reply:
We already do disable allocation after writing the string.